curvefs/src/metaserver/inode_storage.cpp

```diff
@@ -71,5 +84,79 @@ int MemoryInodeStorage::Count() {
 InodeContainerType *MemoryInodeStorage::GetInodeContainer() {
     return &inodeMap_;
 }
+
+void MemoryInodeStorge::MergeTwoS3ChunkInfoMap(
+    const ::google::protobuf::Map<uint64_t, S3ChunkInfoList>& mapA,
```

Comment:
mapA modify new, mapB modify old?

Reply:
no "new“ or ”old“, just merge 2 map and output to result